src/main/java/com/minecolonies/coremod/items/ItemSupplyCampDeployer.java

```diff
@@ -158,9 +162,19 @@ public static boolean canCampBePlaced(@NotNull final World world, @NotNull final
      * @param pos  the position.
      * @return true if is water
      */
-    private static boolean checkIfSolidAndNotInColony(final World world, final BlockPos pos)
+    private static boolean checkIfSolidAndNotInColony(final World world, final BlockPos pos, @NotNull List<PlacementError> placementErrorList)
     {
-        return world.getBlockState(pos).getMaterial().isSolid() && notInAnyColony(world, pos);
+        final boolean isSolid = world.getBlockState(pos).getMaterial().isSolid();
+        final boolean notInAnyColony = notInAnyColony(world, pos);
+        if (!isSolid) {
+            final PlacementError placementError = new PlacementError(PlacementErrorType.NOT_SOLID, pos);
+            placementErrorList.add(placementError);
+        }
+        if (!notInAnyColony) {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Move this left curly brace to the beginning of next line of code. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3ALeftCurlyBraceStartLineCheck)